tests/test_flight_boundary.py

```diff
@@ -0,0 +1,23 @@
+from matplotlib.path import Path
+from SDA import*
+import unittest
+import numpy as np
```

Comment:
Please remove the ```np``` reference. I know this is done a lot in the codebase, but it is a bad habit...